temporalcli/commands.workflow_reset_test.go

```diff
@@ -600,27 +600,18 @@ func (t *WorkflowResetTest) startWorkflowAndSendTwoSignalsAndTwoUpdates(searchAt
 	)
 	s.NoError(err)
 
-	// Wait for the workflow to start before sending signals/updates.
-	// This has to be done, as batch reset with type `FirstWorkflowTask`, will reset to first workflow task completed, so the first signal
-	// sent before the workflow starts, will be reapplied, as the reset point is later in the history.
-	// The same would happen with single reset to eventId 4.
-	s.Eventually(func() bool {
-		resp, err := s.Client.ListWorkflow(s.Context, &workflowservice.ListWorkflowExecutionsRequest{
-			Query: "CustomKeywordField = '" + searchAttr + "'",
-		})
-		s.NoError(err)
-		return len(resp.Executions) > 0
-	}, 3*time.Second, 100*time.Millisecond, "Workflow failed to start")
```

Comment:
This code was waiting for the workflow to exist, whereas we needed to wait for the workflow task be complete.